src/app/globals.css

```diff
@@ -25,3 +25,28 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+h1 {
+  font-size: 2em;
+  margin-bottom: 1rem;
+}
+
+h2 {
+  font-size: 1.5em;
+  margin-bottom: 0.5rem;
+}
+
+p {
+  margin-bottom: 0.5rem;
+}
+
+.centerText {
+  text-align: center;
+}
+
+.page-landing-background {
```

Comment:
[Comment]: Once we move this into a CSS module we won't need a specific name, so just `landing-background`/`background` would do. Alternatively, this could just set the background for `main` directly.

Reply:
I'm choosing to rename it to "landing-background" rather than adding it to main in order to minimize potential for conflicts for now, but if its best to change it to a feature of main in the future then that works with me.

Reply:
Resolved, changed to .landing-background